setup.cfg

```diff
@@ -19,6 +19,7 @@ install_requires =
     six>=1.14.0
     click>=7.0.0
     pip>=10.0.0
+    semver
```

Comment:
Should this have a version spec? The `semver` package has started 3.0.0dev builds. Assuming they are following semver, they will likely release a breaking change soon. We haven't pinned major versions on the other dependencies, but this seems like one we should consider.